Gemfile.lock

```diff
@@ -244,6 +244,7 @@ PLATFORMS
 DEPENDENCIES
```

Comment:
apologies for not being clear on this, or familiar with Ruby. This happened upon recompiling the docker image. Should I remove these? It seems you have a bot that manages this... let me know.

Reply:
Yeah, let's not change `Gemfile.lock`.